pyaerocom/ungriddeddata.py

```diff
@@ -1717,6 +1724,8 @@ def _find_meta_matches(self, negate=None, *filters):
             if self._check_filter_match(meta, negate, *filters):
                 meta_matches.append(meta_idx)
                 for var in meta["var_info"]:
+                    if var == "altitude":
```

Comment:
There's more than altitude as vertical coordinates in meteorology (pressure, sigma, hybrid sigma etc.) I think this should be a list of strings stored as contstant somewhere... (non blocking request)

Reply:
I created a constant in `UngriddedData` called `ALLOWED_VERT_CORD_TYPES` which could be expanded upon in the future.